src/lib/support/verhoeff/tests/BUILD.gn

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+import("//build_overrides/pigweed.gni")
+import("${chip_root}/build/chip/chip_test_suite.gni")
+
+chip_test_suite("tests") {
+  output_name = "libVerhoeffTest"
+
+  test_sources = [ "TestVerhoeff.cpp" ]
+
+  sources = []
+
+  public_deps = [
+    "${chip_root}/src/app/common:cluster-objects",
```

Comment:
this dependency list feels a bit big. Why do we need cluster objects for example? can we clean  it up?

Reply:
I believe you only need to depend on support is sufficient. I believe right now you get `support:suppport` by transitive deps as a sideffect.